packages/babel-plugin-transform-decorators-2/test/fixtures/transformations/method-and-class-decorators/expected.js

```diff
@@ -0,0 +1,6 @@
+let Foo = babelHelpers.decorate(class Foo {
+  method() {}
+
+  undecorated() {}
+
+}, [["undecorated"]], [["method", [methDec]]], void 0)([classDec]);
```

Comment:
currently I'm using `const [key, isStatic] = [..]` so that's why the inner items of those are arrays (in case of a non static member, I just omit the second array element). We could certainly simplify it further if needed, e.g. if it's static then it'll look like `["method", true]` otherwise it'll look like just `"method"` and not `["method"]`